go.mod

```diff
@@ -108,15 +108,15 @@ require (
 	github.com/hashicorp/vault-plugin-secrets-azure v0.11.2
 	github.com/hashicorp/vault-plugin-secrets-gcp v0.11.0
 	github.com/hashicorp/vault-plugin-secrets-gcpkms v0.10.0
-	github.com/hashicorp/vault-plugin-secrets-kv v0.5.7-0.20211123171606-16933c88368a
+	github.com/hashicorp/vault-plugin-secrets-kv v0.5.7-0.20211206210934-74fe79f60b74
```

Comment:
Were these upgrades intentional? If you wanted to include the upgrades in the other 2 PRs open for review, I think you'd need to re-import after those PRs are merged.

Reply:
Yes, the intention is to include the upgrade for `vault-plugin-secrets-kv` for review. As you mentioned, I plan to re-import after those PRs are merged.